presentation-abstraction-control/src/main/java/com/iluwatar/pac/App.java

```diff
@@ -0,0 +1,28 @@
+package com.iluwatar.pac;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class App {
+  /**
+   * The main function.
+   *
+   * @param args Normally no arguments.
+   */
+  public static void main(String[] args) {
+    var studentStatusControl = new StudentStatusControl();
+    var studentCourseControl = studentStatusControl.getStudentCourseControl();
+    studentCourseControl.addCourseGrade("Math", 90.0f);
+    studentCourseControl.addCourseGrade("Physics", 20.0f);
+    studentCourseControl.present();
+    StudentAverageGradeControl studentAverageGradeControl =
+            studentStatusControl.getStudentAverageGradeControl();
+    studentAverageGradeControl.present();
+    studentStatusControl.present();
+    studentCourseControl.changeCourseGrade("Physics", 60.0f);
+    LOGGER.info("After change physics' grade");
+    studentCourseControl.present();
+    studentAverageGradeControl.present();
+    studentStatusControl.present();
```

Comment:
Add comments explaining what's going on

Reply:
One idea is to use a fluent interface style. With small changes we could write something like
`studentStatusControl.getStudentCourseControl().addCourseGrade("Math", 90.0f).addCourseGrade("Physics", 20.0f).present()`

Reply:
Students, courses, and grades are kind of boring. I would love to see something more interesting as the topic of the example...